### PR TITLE
Chore create shell script for generating requirements development lock

### PR DESCRIPTION
**Pull request by kbeker**
Resolves https://github.com/golemfactory/concent/issues/746

### CODE REVIEW

File: generate-requirements-development-lock.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+virtualenv /tmp/generate-requirements-development-lock
+source /tmp/generate-requirements-development-lock/bin/activate
+pip install -r ${BASH_SOURCE%/*}/concent_api/requirements.lock
```
Comment (bartoszbetka) [inline generate-requirements-development-lock.sh]: We settled with @cameel  to use `python -m pip install -r "requirements.lock"` form. What do you think about this @cameel ?

File: generate-requirements-development-lock.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+virtualenv /tmp/generate-requirements-development-lock
```
[the remark this answers is not in the document]
Comment (cameel) [reply generate-requirements-development-lock.sh]: But use just `python3`, not `python3.6`. No need to hard-code more here.

File: generate-requirements-development-lock.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+source /tmp/generate-requirements-development-lock/bin/activate
+pip install -r ${BASH_SOURCE%/*}/concent_api/requirements.lock
+pip install -r ${BASH_SOURCE%/*}/requirements-development.txt
+echo "--find-links https://builds.golem.network/simple/pyelliptic" > ${BASH_SOURCE%/*}/requirements-development.lock
+pip freeze >> ${BASH_SOURCE%/*}/requirements-development.lock
+deactivate
+rm -rf /tmp/generate-requirements-development-lock
```
[the remark this answers is not in the document]
Comment (cameel) [reply generate-requirements-development-lock.sh]: And don't use `--force`.

File: generate-requirements-development-lock.sh (hunk 1 above)
Comment (cameel) [reply generate-requirements-development-lock.sh]: It's a bit safer but it does not matter all that much here. This script is only used in development.

The difference between `pip` and `python -m pip` is only relevant when you have an old version of `pip` installed on the machine. That might be a case in a docker container but it's not all that likely on developer's machine.

**Comment by cameel** [comment]
@kbeker Please see the comments in the issue.

**Comment by kbeker** [comment]
Im closing this issue. After discuss it should be done in different way: https://github.com/golemfactory/concent/pull/781

**Comment by kbeker** [comment]
Im closing this issue. After discuss it should be done in different way: #781

**Pull request closed without merging**